README.md

```diff
@@ -99,15 +102,15 @@ Once this is done, update `config-instance.ts` in `tunnelmole-client` to point t
 ```
 const instanceConfig = {
     hostip: {
-        endpoint: "wss://foo.com:81"
+        endpoint: "ws://foo.com:8080"
```

Comment:
any reason to not use the secure `wss`? it probably would not matter locally but if its exposed over the internet it should be encrypted.

Reply:
It doesn't work locally.